### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by didierganthier**
# In this project we designed the contact form and integrated formspree.

 *I used:*
  - `form` and `input` to handle user inputs
  - `formspree` to receive messages from form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -185,5 +185,15 @@ <h3 class="languages-title">Languages</h3>
             <img src="images/separator.png" class="separator" alt="separator">
         </div>
     </section>
+    <section class="contact">
+        <h1 class="contact-title">Contact me</h1>
+        <p class="contact-subtitle">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+        <form class="form-section" method="post" action="https://formspree.io/f/xdojkarj">
+            <input type="text" placeholder="Enter your name" required maxlength="30"/>
+            <input type="email" name="email" placeholder="Enter your email" required />
+            <textarea placeholder="Write your message here" maxlength="500"></textarea>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Kindly, add `required` attribute for text area validation, All input fields must have the `required` attribute to activate the validation on submit.

File: style.css [hunk 2]
```diff
@@ -242,6 +242,74 @@ hr {
...
+  border: 1px solid #6071fe;
+  padding: 10px;
+  color: #6071fe;
+  margin-top: 10px;
+  background: white;
+  font-weight: bold;
+  font-size: medium;
+  cursor: pointer;
+}
+
```
Comment (Abenezer-Tilahun) [inline style.css]: - [ ] Please try to  follow the Figma design Requirement, You must stick to the design as much as possible this includes the font, colors, images and spacing (margin/padding) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
- [ ] Kindly implement Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
- [ ] kindly try to include background images just like Figma design show below : 
fig12

Your Project Look likes this : 

cont

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @didierganthier  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

almost

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jasonfelice — APPROVED** [review]
Hi @didierganthier, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
Great design✔️
Meaningful commit messages✔️
Followed GitHub flow✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**